cli/cli.go

```diff
@@ -92,6 +96,8 @@ func before(c *cli.Context) error {
 		configs.IsPullRequestMode = true
 	}
 
+	configs.BitriseVersionStr = c.App.Version
```

Comment:
move the version into a `version` package instead https://github.com/bitrise-io/bitrise-webhooks/blob/master/version/version.go